File: Backend/Backend_ProyectoFinal/Backend_ProyectoFinal/Models/Empleado.cs
namespace Backend_ProyectoFinal.Models
{
    public class Empleado
    {
        private Departamento departamento;
        private int idEmpleado;
        private string nombre;
        private string apellido;

        public Departamento Departamento { get => departamento; set => departamento = value; }
        public int IdEmpleado { get => idEmpleado; set => idEmpleado = value; }
        public string Nombre { get => nombre; set => nombre = value; }
        public string Apellido { get => apellido; set => apellido = value; }

        public Empleado(Departamento departamento, int idEmpleado, string nombre, string apellido)
        {
            this.Departamento = departamento;
            this.IdEmpleado = idEmpleado;
            this.Nombre = nombre;
            this.Apellido = apellido;
        }

        public Empleado()
        {
        }
    }
}

File: main/java/com/example/appmovil/model/Empleado.java
package com.example.appmovil.model;

public class Empleado {
    private Departamento departamento;
    private int idEmpleado;
    private String nombre;
    private String apellido;

    public Departamento getDepartamento() {
        return departamento;
    }

    public void setDepartamento(Departamento departamento) {
        this.departamento = departamento;
    }

    public int getIdEmpleado() {
        return idEmpleado;
    }

    public void setIdEmpleado(int idEmpleado) {
        this.idEmpleado = idEmpleado;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public Empleado(Departamento departamento, int idEmpleado, String nombre, String apellido)
    {
        this.departamento = departamento;
        this.idEmpleado = idEmpleado;
        this.nombre = nombre;
        this.apellido = apellido;
    }

    public Empleado()
    {
    }
}
